### ISSUE TITLE
Add creation and visibility information to detail page

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backend#471]
# Story

Show creation date, creator, biblio status and visibility to in record detail page.

Already showing: modification date. Optional: show who updated it as well.

# Context

## Old Biblio

In the old biblio information, this information was shown to researchers, curators and reviewers alike to make sure they have the information they need to contact someone if needed.

![Image](https://user-images.githubusercontent.com/1572885/186098233-db3e8f97-d459-4685-a0f5-3238f22fa9d1.png)

## Current implementation in new Biblio

Some data is missing:

![Image](https://user-images.githubusercontent.com/1572885/186100948-a8516c75-f3a8-4dc9-8085-013a24eb8885.png)

# Solution

To reach feature parity, we will add a bit more data to help out researchers, reviewers and data curators.

![Image](https://user-images.githubusercontent.com/1572885/186098697-c2c07b9e-f317-4382-b6f2-bae1c1788f73.png)

## Success Criteria

Should be implemented for both publications and datasets.

- [x] Show type of record
- [x] Show status (now split up into Biblio status and visibility)
- [x] Creation date
- [x] Creator

## Implementation

View the original code at https://biblio-ugent-prototype-emily-v2.vercel.app/biblio/researcher/publication-detail/description.html. The code below left out information we will add later, we will step-by-step go to this lay-out. There are slight adaptions to make it work visually, but the layout has been respected to the maximum.

## Adapted code:

```
<div class="c-thumbnail-text u-min-w-0">
  <div class="bc-toolbar bc-toolbar--auto flex-column-reverse flex-lg-row align-items-stretch align-items-lg-center mt-n3">
    <div class="bc-toolbar-left overflow-hidden flex-wrap">
      <span class="c-subline text-nowrap mr-3 pr-3 my-2">Journal Article: A1</span>
    </div>
    <div class="bc-toolbar-right">
      <div class="bc-toolbar-item mr-auto mr-lg-0">
        <div class="d-flex align-items-center flex-wrap justify-content-md-end">
          <span class="c-body-small text-muted text-truncate my-2">Created at today at 13:24 by <span class="">
            <a class="c-link c-link-muted" href="#">Jane Cooper</a>
            <i class="if if-ghent-university" data-toggle="tooltip" title="" data-placement="bottom" data-original-title="UGent author"></i>
          </span> – Updated today at 16:24</span>
        </div>
      </div>
    </div>
  </div>
  <h4 class="list-group-item-title">Neuropsychiatric aspects of COVID-19 pandemic: A selective review</h4>
  <div class="bc-toolbar bc-toolbar-responsive flex-wrap-reverse">
    <div class="bc-toolbar-left flex-wrap">
      <div class="bc-toolbar-item my-2">
        <p class="c-body-small text-muted">Draft</p>
      </div>
      <div class="bc-toolbar-item my-2">
        <span class="badge badge-pill badge-default">
          <span class="badge-text">Biblio Public</span>
        </span>
      </div>
    </div>
  </div>
  <div class="u-no-transition mt-5 collapse" id="moreDetails-20393" style="">
    <div class="card">
      <div class="card-body bg-lightest">
        <dl class="c-dl row c-body-small align-items-start px-3">
          <dt class="col-md-2 c-dl-term">Short title</dt>
          <dd class="col-md-10 c-dl-desc">Neuropsychiatric aspects of COVID-19 pandemic: A selective review</dd>
          <dt class="col-md-2 c-dl-term">Authors</dt>
          <dd class="col-md-10 c-dl-desc">
            <div class="c-author-list">
              <div class="c-author">
                <a class="c-link c-link-muted" href="#">Jane Cooper</a>
                <i class="if if-ghent-university" data-toggle="tooltip" title="" data-placement="bottom" data-original-title="UGent author"></i>
              </div>
              <div class="c-author">
                <a class="c-link c-link-muted" href="#">Brooklyn Simmons</a>
                <i class="if if-ghent-university" data-toggle="tooltip" title="" data-placement="bottom" data-original-title="UGent author"></i>
              </div>
              <div class="c-author">
                <a class="c-link c-link-muted" href="#">Esther Howard</a>
                <i class="if if-orcid" data-toggle="tooltip" title="" data-placement="bottom" data-original-title="Author with ORCID"></i>
                <i class="if if-ghent-university" data-toggle="tooltip" title="" data-placement="bottom" data-original-title="UGent author"></i>
              </div>
            </div>
          </dd>
          <dt class="col-md-2 c-dl-term">Creation date</dt>
          <dd class="col-md-10 c-dl-desc">23-03-2022</dd>
        </dl>
      </div>
    </div>
  </div>
</div>
```

**Comment by mietcls** [comment]
Note; think about overview page as well. Probably similar.

**Comment by nics** [comment]
@mietcls the more details content is there, but the button to toggle it is not

**Comment by mietcls** [comment]
@nics ah I think that was a mistake that I left in the code per accident, thanks for mentioning it.

**Comment by mietcls** [comment]
I still have to make an issue for locking for reviewers and the impact on this card, will create a separate issue. I'll take it out of this one, and then I think it can be closed (will double check on eliot)

**Comment by nics** [comment]
@mietcls creator and last updater are just displayed by their id, will be replaced by their name later, need some other building blocks in place to do that

**Comment by nics** [comment]
fixed by d9727694962209c5fd034a45aa0e8a2805ba719d
